prime-router/src/main/kotlin/history/azure/DeliveryFunction.kt

```diff
@@ -203,6 +207,28 @@ class DeliveryFunction(
         }
     }
 
+    @FunctionName("getReportItemsV1")
+    fun getMetadataForReport(
```

Comment:
Might be useful for debugging, but mostly added this as a way for someone to manually test the new query